src/app/fda/application/application-form/can-activate-register-application-form.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { Router, CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, NavigationExtras, UrlTree } from '@angular/router';
+import { AuthService } from '@gsrs-core/auth/auth.service';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class CanActivateRegisterApplicationFormComponent implements CanActivate {
+
+  constructor(
+    private router: Router,
+    private authService: AuthService
+  ) {}
+
+  canActivate(
+    route: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot
+  ): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | (boolean | UrlTree) {
+    return new Observable(observer => {
+      this.authService.getAuth().subscribe(auth => {
```

Comment:
Can you close off these two authService subscriptions with pipe(take(1)) just to be safe?